packages/arb-token-bridge-ui/src/components/TokenModal/TokenModalUtils.ts

```diff
@@ -0,0 +1,147 @@
+import { useMemo } from 'react'
+import { ERC20BridgeToken, TokenType } from 'token-bridge-sdk'
+
+import { useAppState } from '../../state'
+import { TokenListWithId, useTokenLists } from '../../tokenLists'
+
+export interface SearchableToken extends ERC20BridgeToken {
+  tokenLists: number[]
+}
+
+export type SearchableTokenStorage = { [address: string]: SearchableToken }
```

Comment:
the `address` key is always the L1 address, right? could we call it `l1Address` or leave a comment here specifying this